### PR TITLE
Tweak init logic, set parent container height

### PR DESCRIPTION
**Pull request by gashcrumb**
This makes sure the data mapper component has the right height so the left/right panels scroll individually.

### CODE REVIEW

**Comment by gashcrumb** [comment]
lemme also push a commit to update the datamapper component before merging this

**Comment by gashcrumb** [comment]
@kahboom k, ready to merge!

File: src/app/integrations/edit-page/step-configure/data.mapper.example.host.component.ts [hunk 1]
```diff
@@ -135,27 +141,35 @@ export class DataMapperHostComponent extends FlowPage implements OnInit, OnDestr
...
+          this.initializeMapper();
+        }, 10);
       },
     });
   }
 
   initializeMapper() {
+    this.cfg.mappingService.mappingUpdated$.subscribe(() => {
+      this.detector.detectChanges();
+    });
```
Comment (kahboom) [inline src/app/integrations/edit-page/step-configure/data.mapper.example.host.component.ts]: This lack of spaces between functions is driving me nuts. :see_no_evil:

Comment (kahboom) [inline src/app/integrations/edit-page/step-configure/data.mapper.example.host.component.ts]: If it bothers you too, we should consider updating the lint config.. although I do agree with forcing a new line at the end of each file.

**Pull request merged**